wp-content/themes/core/base.twig

```diff
@@ -1,15 +1,14 @@
 {% block header %}
-	{{ include( 'header.twig' ) }}
+	{{ header }}
 {% endblock %}
 
 <a id="a11y-skip-link-content" tabindex="-1"></a>
 
 <main>
-
 	{% block main %}
 
 		{% block subhead %}
-			{{ include( 'content/header/sub.twig' ) }}
```

Comment:
Ooooh is the goal here to move away from using twig `include()` and always just pass sub-templates from the Controller? I like that.